src/Controller/DashboardController.java

```diff
@@ -56,10 +89,12 @@
 				}
 			}
 		}
-		printToConsole("ERROR: Did not find any user with the entered name:" + loginName.getText() + ".");
+		printToConsole("ERROR: Did not find any user with the entered name (" + loginName.getText() + ") and password.");
+		loginName.setText("");
+		loginPassword.setText("");
 	}
 
-	@FXML private void addUser() {
+	@FXML private void createUser() {
 		UserType type = null;
 		if (createUserName.getText().length() == 0) {
```

Comment:
as I said in another comment.  Maybe we should refactor to move this logic to the the Simulation class.  This is low priority though.